docs/js-guide/requirejs-to-webpack.rst

```diff
@@ -0,0 +1,214 @@
+like the ones listed below. We want to migrate the RequireJS entry points to Webpack entry points:
+
+ - ``requirejs_main_b5`` to ``webpack_main``
+ - ``requirejs_main`` to ``webpack_main_b3``
+
+Entry points are the modules that the bundler (RequireJS or Webpack) uses to build a dependency graph so
+that it knows what bundle of javascript dependencies and page-specific code is needed to render that page.
+See `the Static Files Overview <https://github.com/dimagi/commcare-hq/blob/master/docs/js-guide/static-files.rst>`__
+for a more detailed explanation.
+
+Step 1: Identify the Entry Point and Details
+--------------------------------------------
+
+First, identify the entry point you would like to migrate. Additionally, we need to determine if the entry
+point is split along the Bootstrap 3 or Bootstrap 5 RequireJS build.
+
+Bootstrap 5 Entry Points
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+As an example, let's migrate ``case_importer/js/main``.
+
+We should find a ``requirejs_main_b5`` template tag referencing this entry point inside a template.
+
+The following template tag is present in ``case_importer/excel_config.html``:
+
+::
+
+    {% requirejs_main_b5 "case_importer/js/main" %}
+
+Since the ``requirejs_main_b5`` template tag is being used, we know this entry point is part of the Bootstrap 5
+build. We also do not need to be concerned that this file might be undergoing a Bootstrap 5 migration.
+
+We can now proceed to Step 2.
+
+Bootstrap 3 Entry Points
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+As an example, let's migrate ``domain/js/my_project_settings``.
+
+We should find a ``requirejs_main`` template tag referencing this entry point inside a template.
+
+The following template tag is preset in ``domain/admin/my_project_settings.html``:
+
+::
+
+    {% requirejs_main "domain/js/my_project_settings" %}
+
+Since the ``requirejs_main`` template tag is being used, we know this entry point has not been migrated
+from Bootstrap 3 to 5. Since we are currently undergoing a migration from Bootstrap 3 to 5, it is important
+to establish that this page is not actively undergoing a migration.
+
+Please see the `Bootstrap Migration Status List
+<https://docs.google.com/spreadsheets/d/1tkSXR643Da-fp6a-uYPa5dYs5if4W2LqtvUJs3IfUKs/edit?gid=0#gid=0>`__
+to see if the application housing that entry point is undergoing a migration. If unsure, please
+raise a question to ``#gtd-dev``.
+
+If you are able to determine that this entry point is NOT actively undergoing a Bootstrap 3 to 5 migration,
+then please proceed to Step 2.
+
+
+Step 2: Update the Template Tag and Add Global ``commcarehq`` Dependency
+------------------------------------------------------------------------
+
+The migration of an entry point from RequireJS to Webpack will involve updating the template tag
+used to define the entry point and then adding the ``commcarehq`` global dependency to the list of dependencies.
+
+Bootstrap 5 Entry Points
```

Comment:
I think it'd be simpler to combine this and the next section, and just say if the page uses `requirejs_main_b5`, replace with `webpack_main`, whereas if it uses `requirejs_main` replace it with `webpack_main_b3`.

Reply:
did some rewording